### DISCUSSION TITLE
Optional confirmation of selections in the menu with the middle button.

### DISCUSSION THREAD

**Discussion by Mike86-CZE** [PE5PVB/TEF6686_ESP32#253] [Ideas]
I have a silver Chinese TEF and it often takes a lot of force to press the rotary controller to get it to respond.  I would like to be able to confirm menu selections with the middle button on the front.  Submenus and the main menu would then be exited by a long press of the middle button.
Alternatively, the first or third button can be used to return.  In the menu, these buttons have no function.

**Comment by PE5PVB** [comment]
The menu button will act like the rotary button when a menu item is open.

**Comment by andimik** [comment]
I had a similar situation, but a (propriate) screwdriver helped here, in case you have that version with screws.

![_20230820_080446](https://github.com/PE5PVB/TEF6686_ESP32/assets/24510556/f74950de-69f8-4e22-8a5c-b4f35ff1c3f8)

If not, you can easily remove them by pulling them off.

**Comment by Mike86-CZE** [comment]
Confirmation with the button on the front seems more practical to me.  It would be better controlled with one hand.

**Comment by HyperDX** [comment]
> I have a silver Chinese TEF and it often takes a lot of force to press the rotary controller to get it to respond.

This could mean that your rotary encoder need to be replaced shortly.

**Reply by Mike86-CZE** [reply]
Rather, it sometimes rotates to an intermediate position when pressed.  It's just not convenient for frequent confirmation of choices.

**Comment by PE5PVB** [comment]
This is hardware related.
